### PR TITLE
Add LLBLGen Pro Info

### PR DESCRIPTION
**Pull request by rowanmiller**
Added Tools & Extensions section and info about LLBLGen Pro

### CODE REVIEW

**Comment by rowanmiller** [comment]
@divega could you take a quick look over this

**Comment by bradygaster** [comment]
### :white_check_mark: Validation status: passed


For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C1%5C11%5Cc1705ea6-142e-cb02-f3b5-8dd98c6d1733%5CPullRequest%5C201701112205345090-326%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: entity-framework/core/extensions/llbl-gen-pro.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+# LLBLGen Pro
+
+> [!NOTE]
+> This documentation is for EF Core. For EF6.x, see [Entity Framework 6](../../ef6/index.md).
+
+> [!NOTE]
+> This extension is not maintained as part of the Entity Framework Core project. When considering a third party extension, be sure to evaluate quality, licensing, support, etc. to ensure they meet your requirements.
+
+LLBLGen Pro is an entity modeling solution for Entity Framework and Entity Framework Core. It lets you easily define your entity model and map it to your database, using database first or model first, so you can get started writing queries right away.
```
Comment (divega) [inline entity-framework/core/extensions/llbl-gen-pro.md]: Should we use "entity modeling solution with support for"? It seems that other O/RMs as well.

**Comment by ErikEJ** [comment]
I maintain a blog post of EF Core extension libraries that are available on Nuget - would that be useful to add to the docs? http://erikej.blogspot.dk/2016/12/entity-framework-core-extensions.html

**Comment by bradygaster** [comment]
### :white_check_mark: Validation status: passed


For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C1%5C13%5Cc1705ea6-142e-cb02-f3b5-8dd98c6d1733%5CPullRequest%5C201701131811227556-326%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by rowanmiller** [comment]
@ErikEJ are you offering a PR to the new extensions section? 😄 Yes, I think it would be good to list them.

**Comment by ErikEJ** [comment]
Sure, I will see if I can figure it out!

**Pull request merged**